CIPs/cip-0045.md

```diff
@@ -11,32 +11,39 @@ license: Apache 2.0
 ---
 
 ## Simple Summary
-Paying transaction fees using non-native currencies should be feasible if a transactor has a balance greater than or equal to the fee.
+Paying transaction fees using non-native currencies should be feasible if send's balance >= fee + value.
```

Comment:
I prefer the old wording here. ("send's" is grammatically incorrect & I don't think transactor is confusing).
I also prefer the written math over symbols for this summary.